### ISSUE TITLE
Lack of visual separation between code editing panel and log

### ISSUE THREAD

**Issue by jeanas** [frescobaldi/frescobaldi#1823]
I get this appearance on master (commit 74ad54722d55c53c992a82041d226c462cebf94f):

![Capture d’écran du 2024-11-24 16-54-22](https://github.com/user-attachments/assets/63d64f88-746d-47a0-ae2a-31150db2edc0)

<details>
<summary>OS info:</summary>
Frescobaldi: 3.3.0
Extension API: 0.9.0
Python: 3.13.0
python-ly: 0.9.7
Qt: 6.8.0
PyQt: 6.8.0.dev2410141303
qpageview: 0.6.2
poppler: unknown
python-poppler-qt: unknown
OS: Linux-6.11.7-300.fc41.x86_64-x86_64-with-glibc2.40 (Fedora Linux 41 (Workstation Edition))
Installation kind: distro package or installed from source
</details>

**Comment by fedelibre** [comment]
@jeanas I'm trying to run Frescobaldi master from git on Fedora 41 (same as you) for the first time and I'm getting this error:

```
$ ./frescobaldi
Traceback (most recent call last):
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/plugin.py", line 79, in instance
    return _instances[cls][obj]
           ~~~~~~~~~~^^^^^
  File "/usr/lib64/python3.13/weakref.py", line 415, in __getitem__
    return self.data[ref(key)]
           ~~~~~~~~~^^^^^^^^^^
KeyError: <weakref at 0x7fdeab7bdcb0; to 'type' at 0x563b1bc9a540 (PanelManager)>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/./frescobaldi", line 30, in <module>
    main()
    ~~~~^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/__main__.py", line 231, in main
    win = mainwindow.MainWindow()
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/mainwindow.py", line 142, in __init__
    self.createMenus()
    ~~~~~~~~~~~~~~~~^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/mainwindow.py", line 1129, in createMenus
    menu.createMenus(self)
    ~~~~~~~~~~~~~~~~^^^^^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/menu.py", line 61, in createMenus
    m.addMenu(menu_file(mainwindow))
              ~~~~~~~~~^^^^^^^^^^^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/menu.py", line 95, in menu_file
    m.addMenu(snippet.menu.TemplateMenu(mainwindow))
              ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/snippet/menu.py", line 149, in __init__
    self.addAction(self.tool().actionCollection.templates_manage)
                   ~~~~~~~~~^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/snippet/menu.py", line 58, in tool
    return panelmanager.manager(self.mainwindow()).snippettool
           ~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/panelmanager.py", line 38, in manager
    return PanelManager.instance(mainwindow)
           ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/plugin.py", line 84, in instance
    result.__init__(obj)
    ~~~~~~~~~~~~~~~^^^^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/panelmanager.py", line 70, in __init__
    self.loadPanel("musicview.MusicViewPanel", "viewers")
    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/panelmanager.py", line 111, in loadPanel
    panel = cls(self.mainwindow())
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/musicview/__init__.py", line 91, in __init__
    ac = self.actionCollection = Actions(self)
                                 ~~~~~~~^^^^^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/actioncollection.py", line 138, in __init__
    self.createActions(parent)
    ~~~~~~~~~~~~~~~~~~^^^^^^^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/musicview/__init__.py", line 219, in createActions
    self._viewActions = va = pagedview.ViewActions()
                             ~~~~~~~~~~~~~~~~~~~~~^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/pagedview.py", line 239, in __init__
    super().__init__(*args, **kwargs)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/site-packages/qpageview/viewactions.py", line 70, in __init__
    self.setActionIcons()
    ~~~~~~~~~~~~~~~~~~~^^
  File "/mnt/dati/home/fede/src/frescobaldi/frescobaldi/frescobaldi_app/pagedview.py", line 249, in setActionIcons
    self.print.setIcon(icons.get('document-print'))
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: setIcon(self, icon: QIcon): argument 1 has unexpected type 'QIcon'
```

Do you know what I may missing?

**Comment by fedelibre** [comment]
Here's my screenshot:

![immagine](https://github.com/user-attachments/assets/662caf2d-0fda-4d24-adef-eff4fe38b9eb)

**Comment by jeanas** [comment]
I'm on Fedora 41 with GNOME, default flavor of Fedora (Workstation Edition), no customizations at all (in fact I didn't upgrade from Fedora 40 but installed the system afresh).

But here's what I noticed now:

https://github.com/user-attachments/assets/06b3177b-6022-429d-8bd1-48baa0317793

Actually, the appearance is the same as on @fedelibre's desktop, except that the very thin separations sometimes vanish, which is probably a rendering issue.

I got a similar issue in my text editor with Ptyxis, the new default terminal in Fedora: https://gitlab.gnome.org/chergert/ptyxis/-/issues/264 https://github.com/helix-editor/helix/discussions/12025 (the first one is on GNOME's GitLab instance which looks down at the moment). The Ptyxis maintainer replied that this was not an issue in Ptyxis but I didn't understand the technical details.
